pyproject.toml

```diff
@@ -13,8 +13,7 @@ build-backend = "setuptools.build_meta"
 
 [tool.pytest.ini_options]
 # Raise warnings to exceptions.
-filterwarnings = 'error'
-
+filterwarnings = ["error", "default::DeprecationWarning", "default::FutureWarning"]
```

Comment:
Nice.  This is a great idea and I'm glad they're supporting this in `pyproject.toml` now.